src/types/__tests__/profile.test.ts

```diff
@@ -127,4 +127,15 @@ describe("profile type", () => {
       expect(userDataKO._tag).toBe("Left");
     }
   });
+  it("should get an app Proxy profile user profile with accepted_tos_version undefined ", async () => {
+    // return app Proxy Profile.
+    delete mockedExtendedProfile.accepted_tos_version;
```

Comment:
this will affect all tests run after this, do not modify `mockedExtendedProfile`

Reply:
Ok, I will define a specific mocked user for this kind of test.